lib/susedistribution.pm

```diff
@@ -570,6 +571,7 @@ sub console_nr {
     my $nr = 4;
     $nr = get_root_console_tty if ($name eq 'root');
     $nr = 5 if ($name eq 'log');
+    $nr = 3 if ($name eq 'extra');
```

Comment:
Can we try to use backward mapping from the definition in init_consoles?

Reply:
@okurz how would you do that?